freedomofpress/pressfreedomtracker.us
# Fix lead_graphic on special blog posts and blog cards

## Description

Fixes the `lead_image` in the special blog post to use `lead_graphic` and support bar charts, tree maps, etc.

## Type of change



## Testing

- Run this branch locally, and convert a blog post to a special blog post.
- Add a lead graphic as an image, save and make sure that the image appears.
- Switch the lead graphic to a chart, save and make sure that the chart appears.
- Make sure that the special blog card renders properly with the images and charts.

## Checklist

### General checks


### If you made changes to blog

- [x] Verify that the blog index page renders correctly
- [x] Verify that the individual blogs show all the informations correctly

### If you made changes to shared templates (e.g. card design, lead media, etc.)

- [x] Verify that it renders correctly in homepage, if applicable
- [x] Verify that it renders correctly in incident index page, if applicable
- [x] Verify that it renders correctly in individual incident page, if applicable
- [x] Verify that it renders correctly in blog index page, if applicable
- [x] Verify that it renders correctly in individual blog page, if applicable

### If you made any frontend change

![Screenshot 2023-11-07 at 3 51 19 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/b3e247a6-286e-4c3c-af69-2197e47a05e0)
![Screenshot 2023-11-07 at 3 52 26 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/57add674-b1e7-4df5-85ef-912fb3efbdaf)

![Screenshot 2023-11-07 at 4 30 21 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/5fbe920a-fd9c-4508-81b3-d470ccad6be5)
![Screenshot 2023-11-07 at 4 28 47 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/7afe63d5-384c-4554-9ffa-167bb89b3e24)
